package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "intro-storybook-react-template",
+  "version": "0.1.0",
+  "description": "Starter template to get up and running quickly with React and Storybook",
+  "author": "Chromatic <https://chromatic.com/>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chromaui/intro-storybook-react-template"
+  },
+  "bugs": {
+    "url": "https://github.com/chromaui/intro-storybook-react-template/issues"
+  },
+  "license": "MIT",
```

```suggestion
  "name": "Webpack with Storybook and React",
  "version": "0.1.0",
  "description": "A practice with Webpac, Storybook and React",
```

We don't want to inherit the package details of other projects.

---

I had to make all words in the name property lowercase with no spaces because yarn says there are illegal characters.